### PR TITLE
Add styling for top half of new query/query detail view

### PR DESCRIPTION
**Pull request by abbas123456**
Screenshot 2020-05-07 at 14 54 58

### CODE REVIEW

File: core/templates/explorer/query.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                <label class="govuk-label" for="id_connection">
+                    Description
+                </label>
+                <textarea class="govuk-textarea" id="id_description" name="description" cols="40" rows="2"  {% if not can_change %}disabled{% endif %}>{{ form.description.value|default_if_none:"" }}</textarea>
+            </div>
+            <input type="hidden" id="id_created_by_user" name="created_by_user" value="{{ form.created_by_user_email }}" />
+            {% if form.sql.errors %}{% for error in form.sql.errors %}
+                <div class="alert alert-danger">{{ error|escape }}</div>
+            {% endfor %}{% endif %}
+            <div class="panel panel-default sql-panel">
```
Comment (abbas123456) [inline core/templates/explorer/query.html:105]: From this point onwards it is the same as the original query.html from django-sql-explorer

File: core/templates/explorer/query.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
+{% extends 'core/base.html' %}
+
+{% load explorer_tags %}
+
+{% block sql_explorer_navlinks %}
+    {% if can_change %}
+      <li{% if not query %} class="active" {% endif %}><a href="{% url "query_create" %}">New Query</a></li>
+      <li><a href="{% url "explorer_playground" %}">Playground</a></li>
```
Comment (marcuspp) [inline core/templates/explorer/query.html:8]: If you wouldn't mind could all urls be changed so the quotes are double on the outside and single inside href="{% url 'explorer_playground' %}"

Comment (abbas123456) [reply core/templates/explorer/query.html]: done

File: core/templates/explorer/query.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
+{% extends 'core/base.html' %}
+
+{% load explorer_tags %}
+
+{% block sql_explorer_navlinks %}
+    {% if can_change %}
+      <li{% if not query %} class="active" {% endif %}><a href="{% url 'query_create' %}">New Query</a></li>
```
Comment (marcuspp) [inline core/templates/explorer/query.html:7]: indent should be 4

File: core/templates/explorer/query.html (hunk 1 above)
Comment (marcuspp) [reply core/templates/explorer/query.html:105]: If this is going to get replaced maybe put it into a separate file and just add an include for it then it should be easier to replace.

**Pull request merged**